plugins/BEdita/Core/src/ORM/Locator/TableLocator.php

```diff
@@ -52,12 +59,14 @@ protected function _getClassName($alias, array $options = [])
             return $className;
         }
 
-        try {
-            $objectTypes = $this->get('ObjectTypes');
-            $objectType = $objectTypes->get($alias);
-            $options['className'] = $objectType->table;
-        } catch (\Exception $e) {
-            $this->log($e->getMessage(), 'warning');
+        if (!in_array($alias, $this->excludeAlias)) {
```

Comment:
`if (substr($alias, 0, 1) !== '_')`, maybe? We could then consider all aliases starting with `_` as "reserved", even though this should be quite rare.